.gitignore

```diff
@@ -2,3 +2,4 @@ build/
 .gradle/
 testing/
 .idea/php.xml
+.idea
```

Comment:
chore: Please remove the .idea folder from gitignore

It is intended to keep the meta files in the repo.

Reply:
I added that as a catchall because the .xml files were being changed depending on the plugins and what JDK's are being used, will change back.